hooks/image/54-ek-launchers

```diff
@@ -0,0 +1,8 @@
+# Expose Endless Key launchers in the app grid (once a specific content pack has been downloaded)
+
+cat <<- 'EOF' > "${OSTREE_DEPLOYMENT}/etc/systemd/user-environment-generators/62-endless-key.sh"
```

Comment:
The test build failed because this directory doesn't exist. 

```suggestion
mkdir -p "${OSTREE_DEPLOYMENT}/etc/systemd/user-environment-generators"
cat <<- 'EOF' > "${OSTREE_DEPLOYMENT}/etc/systemd/user-environment-generators/62-endless-key.sh"
```